data/dataloader.py

```diff
@@ -1,11 +1,11 @@
 from typing import Tuple
 import pandas as pd
-from type import GlobalPreprocessConfig
+from type import PreprocessConfig
```

Comment:
there's no "preprocessing" now , as we do that within the pipeline, by composing transformations (which is good!) so we can simplify the name of this.